Index: makefile
===================================================================
--- makefile
+++ makefile	2019-02-08 20:33:30.698620998 +0100
@@ -33,7 +33,6 @@
 	install -m 0755 -D -T $(BUILDDIR)libstrangle64.so $(DESTDIR)$(LIB64_PATH)/libstrangle.so
 	install -m 0755 -D -T $(SOURCEDIR)strangle.sh $(DESTDIR)$(bindir)/strangle
 	install -m 0644 -D -T COPYING $(DESTDIR)$(DOC_PATH)/LICENSE
-	ldconfig
 
 clean:
 	rm -f $(BUILDDIR)libstrangle64.so
